src/components/user-list.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import '../css/app.css';
+import UserStore from '../stores/user-store';
+import { addNewUser } from '../actions/user-action';
+import CHANGE_EVENT from '../stores/event-type';
+import UserInfo from '../stores/user-info';
+
+import User from './user.js';
+
+export default class UserList extends React.Component {
+    constructor (props) {
+        super(props);
+
+        this.state = this._getAppState();
+
+        this.handleClickAddUser = this.handleClickAddUser.bind(this);
+        this.handleClickSubmit  = this.handleClickSubmit.bind(this);
+    }
+
+    _getAppState () {
+        return {
+            wantAdd: false,
+            users:   UserStore.getUsers(),
+            size:    UserStore.getSize()
+        };
+    }
+
+    handleClickAddUser () {
+        this.setState({ wantAdd: !this.state.wantAdd });
+    }
+
+    handleClickSubmit () {
+        //TODO: should be rewritten
+        let companyInfo, addressInfo = {};
```

Comment:
we will get an error here, because we assign only the last variable with a default value